styles.css

```diff
@@ -822,15 +822,14 @@
 .path-mod-studentquiz #categoryquestions .header.state {
     text-align: center;
 }
-/* CSS style for message validation form, because the align-items-center is set important in the parent of form-group. */
-@media only screen and (min-width: 396px) {
-    .path-mod-studentquiz #form-advanced-div .form-control-feedback.invalid-feedback {
-        position: absolute;
-    }
-}
 
-@media only screen and (min-width: 576px) {
-    .path-mod-studentquiz #form-advanced-div .form-control-feedback.invalid-feedback {
-        margin-top: 4.5em;
-    }
+/* Specific CSS override align-items-center, we need to prevent align-items: center in the form-group. */
+#fgroup_id_rate_grp div.align-items-center,
+#fgroup_id_rate_grp div.align-items-center,
+#fgroup_id_difficultylevel_grp div.align-items-center,
+#fgroup_id_publiccomment_grp div.align-items-center,
+#fgroup_id_myrate_grp div.align-items-center,
+#fgroup_id_myattempts_grp div.align-items-center,
+#fgroup_id_mydifficulty_grp div.align-items-center {
```

Comment:
We must make sure this style rule does not break layout in other plugins.

Each selector should start with .path-mod-studentquiz or similar (assuming that we really cannot avoid having this.)